exercices/part3/practice1.py

```diff
@@ -0,0 +1,12 @@
+
+
+if __name__ == "__main__":
+    """Write a program that counts for the user. Let the user enter the starting number, the ending number, 
+    and the amount by which to count."""
+
+    start = int(input("Starting number:  "))
+    end = int(input("Ending number: "))
+    count = int(input("Amount by which to count: "))
```

Comment:
After reading the end value, it should be checked that end > start otherwise to re-enter end or close program

Reply:
ok, i will add the check, thank you for the hint